index.html

```diff
@@ -1,183 +1,82 @@
-
-                        <li>That can generate data from this profile via web page, CLI or Rest API interface, configured for volume and output needs.</li>
-
-                        <li>That can validate the data produced back against the profile used to create it</li>
-                    </ul>
-
-                    <h2><a href="https://github.com/finos/datahelix/releases/">Latest release</a></h2>
-
-                    <h2><a href="https://github.com/finos/datahelix/blob/master/docs/DeveloperGuide.md#Contributing">Contributing</a></h2>
-                    
-                    <h2><a id="license"></a>License</h2>
-                    <p>Copyright 2019 Scott Logic Ltd</p>
-                    <p>Distributed under the <a href="http://www.apache.org/licenses/LICENSE-2.0" rel="nofollow">Apache License, Version 2.0</a>.</p>
-                    <p>SPDX-License-Identifier: <a href="https://spdx.org/licenses/Apache-2.0" rel="nofollow">Apache-2.0</a></p>
-
-                </div>
-            </div>
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <title>DataHelix Generator – FINOS</title>
+    <meta
+      name="description"
+      content="Quickly generate customised, complex data for system testing and validation."
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="https://unpkg.com/bootstrap@4.3.1/dist/css/bootstrap.min.css"/>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!--[if lt IE 8]>
+      <p class="browserupgrade">
+        You are using an <strong>outdated</strong> browser. Please
+        <a href="http://browsehappy.com/">upgrade your browser</a> to improve
+        your experience.
+      </p>
+    <![endif]-->
+
+    <nav class="navbar navbar-expand-lg navbar-light bg-light sticky-top">
+      <div class="container">
+        <a class="navbar-brand" href="#">DataHelix Generator</a>
+        <ul class="navbar-nav ml-auto">
+          <li class="nav-item">
+            <a class="nav-link" href="https://github.com/finos/datahelix/">GitHub</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="/playground/">Playground</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="https://github.com/finos/datahelix/blob/master/docs/GettingStarted.md"
+              >Getting Started</a
+            >
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="https://github.com/finos/datahelix/blob/master/docs/UserGuide.md">User Guide</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+
+    <div class="jumbotron">
+      <h1 class="icon-header title">DataHelix Generator</h1>
+      <p>
+        Quickly generate rich and realistic data for simulation and testing.
```

Comment:
Have you considered dropping the fullstop here? To me it looks a tiny bit punchier without it but there is not much in it.